editor/components/document-outline/check.js

```diff
@@ -12,7 +12,7 @@ import { getBlocks } from '../../store/selectors';
 function DocumentOutlineCheck( { blocks, children } ) {
 	const headings = filter( blocks, ( block ) => block.name === 'core/heading' );
 
-	if ( headings.length <= 1 ) {
+	if ( headings.length < 1 ) {
```

Comment:
Minor: This could imply that we'd expect negative, but would never happen, when truly we're interested in checking that the length is explicitly `0`, either:

```js
headings.length === 0
! headings.length
```